builds.expected

```diff
@@ -12077,12 +12077,579 @@ ocurrent/opam-staging:debian-unstable-ocaml-5.2-amd64 -> ocaml/opam:debian-unsta
 	COPY --link [ "Dockerfile", "/Dockerfile.ocaml" ]
 
 ocurrent/opam-staging:debian-unstable-ocaml-5.3-amd64 -> ocaml/opam:debian-unstable-ocaml-5.3
-fedora-37
-fedora-37/arm64
+fedora-38
```

Comment:
This is a big diff to be resulting from a dependency update! :D 

Are we meant to review this diff to look for any particular sorts of problems? Or, if not, what is the utility of this `.expected` file?

Reply:
The `.expected` file is generated by the dummy ocurrent driver within the software.  `dune runtest --auto-promote` updates the file and shows you the resulting diff.  It outputs all of the Dockerfiles, which will be used to build all the base images.  It's a very useful sanity check before deployment that the images will be right.

Reply:
I see. Do you (or should I?) read thru each recipe to ensure it is correct? Or what is the process for catch errors via the expect testing here? No need to answer here. We can talk about it tomorrow brefily perhaps :)

Reply:
I check the diff when I run `--auto-promote`.